index.html

```diff
@@ -14,7 +14,7 @@
 
         <div class="page-heading">How Istanbul works</div>
         <p>
-          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
+          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
```

Comment:
Nope ES2016 has been standardised.

On 3 Aug 2016 20:20, "Kent C. Dodds" <notifications@github.com> wrote:

> In index.html
> <https://github.com/istanbuljs/istanbuljs.github.io/pull/10#discussion_r73402752>
> :
>
> > @@ -14,7 +14,7 @@
> >
> >          <div class="page-heading">How Istanbul works</div>
> >          <p>
> > -          Istanbul instruments your ES5 and ES6 JavaScript code with line counters,
> > +          Istanbul instruments your ES5 and ES2016+ JavaScript code with line counters,
>
> Should be ES2015+
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/istanbuljs/istanbuljs.github.io/pull/10/files/feaed766c653c92d8a451b8ef5e8e8c49b371024#r73402752>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAZQTJBZlTZMMQIPxReuVeJaRQ1YlJ9zks5qcOn9gaJpZM4Jb-yd>
> .
>